tests/test_meshtastic_utils.py

```
@@ -1182,6 +1224,174 @@ def test_connect_meshtastic_existing_client_simple(self):
+        self.assertEqual(observed_rooms, config["matrix_rooms"])
+        self.assertIs(observed_config, config)
+
+
+class TestBleHelperFunctions(unittest.TestCase):
+    """Test BLE helper functions."""
+
+    def test_scan_for_ble_address_find_device_with_timeout(self):
+        """Cover BleakScanner.find_device_by_address(timeout=...)."""
+        from mmrelay.meshtastic_utils import _scan_for_ble_address
+
+        async def _find_device(_address: str, timeout: float | None = None):
```

Ruff (0.14.13)

Unused function argument: `timeout`

(ARG001)